src/lib/components/container.svelte

```diff
@@ -1,24 +1,38 @@
 <script lang="ts">
 	export let direction: 'row' | 'column' = 'column'
 	export let gap = 0
+	export let variant: 'space-between' | 'flex-start' | 'flex-end' | 'center' = 'flex-start'
 </script>
 
-<div class={`root ${direction}`} style={`gap: ${gap}px`}>
+<div class={`container ${direction} ${variant}`} style={`gap: ${gap}px`}>
 	<slot />
 </div>
 
 <style>
-	.root {
+	.container {
 		display: flex;
+		justify-content: flex-start;
 		padding: 0px var(--spacing-24);
 		width: 100%;
-		height: 100%;
-		flex-grow: 1;
 	}
 	.row {
 		flex-direction: row;
 	}
 	.column {
 		flex-direction: column;
 	}
+	.space-between,
+	.stretch,
+	.center {
+		height: 100%;
+	}
```

Comment:
Why `flex-start` and `flex-end` don't have 100% height? It actually seems that these options are ignored and instead there is `stretch`

Reply:
I just pointed this out for consistency reasons.

Reply:
Checking this out now, it looks like adding 100% to flex-start and flex-end does cause issues but I think what I need to do is restructure some other elements/components so I'll do that. 😉